pyfakefs/fake_filesystem.py

```diff
@@ -2034,7 +2034,8 @@ def make_string_path(self, path: AnyPath) -> AnyStr:
     def create_dir(
         self, directory_path: AnyPath, perm_bits: int = helpers.PERM_DEF
     ) -> FakeDirectory:
-        """Create `directory_path`, and all the parent directories.
+        """Create `directory_path` and all the parent directories, and return
+        the created FakeDirectory object.
```

Comment:
Just wondering, why do you sometimes use `code` highlighting (`directory_path`) and sometimes not (FakeDirectory)?

Reply:
> Just wondering, why do you sometimes use `code` highlighting (`directory_path`) and sometimes not (FakeDirectory)?

No reason. I actually was looking for some style guide as I wasn't sure if better to use single or double back ticks, but  got distracted and forgot about it..  Will have another look tomorrow.

Reply:
Shall be fixed now.